.github/workflows/maven.yml

```diff
@@ -5,20 +5,30 @@ name: Java CI with Maven
 
 on:
   push:
-    branches: [ main ]
+    branches:
+      - main
+      - postgres
   pull_request:
-    branches: [ main ]
+    branches: 
+      - main
+      - postgres
 
 jobs:
   build:
 
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
+    - name: Checkout
+      uses: actions/checkout@v4
     - name: Set up JDK
-      uses: actions/setup-java@v1
+      uses: actions/setup-java@v4
       with:
         java-version: 17
+        distribution: 'temurin'
```

Comment:
Could we specify SAP Machine here?

Reply:
no, not available: https://github.com/actions/setup-java?tab=readme-ov-file#supported-distributions
I checked it already, I would also prefer to use SAPMachine